modules/snpsift/split/meta.yml

```diff
@@ -0,0 +1,43 @@
+name: snpsift_split
+description: Splits a VCF file into chromosomes
+keywords:
+  - split
+  - vcf
+tools:
+  - snpsift:
+      description: SnpSift is a toolbox that allows you to filter and manipulate annotated files
+      homepage: https://pcingola.github.io/SnpEff/ss_introduction/
+      documentation: https://pcingola.github.io/SnpEff/ss_introduction/
+      tool_dev_url: https://github.com/pcingola/SnpEff
+      doi: "10.3389/fgene.2012.00035"
+      licence: ['MIT']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
+      description: VCF file
```

Comment:
according to the test gz files also works. Can you add this to the docu? In general we are trying to enforce using compressed files as much as possible. I guess there is no change in the tool whether you use one or the other, right?

Reply:
😢 for uncompressed output. oh well 🤷‍♀️